chainstate/tx-verifier/src/transaction_verifier/mod.rs

```diff
@@ -403,9 +463,15 @@ where
             },
         )?;
 
-        let max_allowed_outputs_total =
-            amount_sum!(inputs_total, block_subsidy_at_height.0, total_fees.0)
-                .ok_or_else(|| ConnectTransactionError::RewardAdditionError(block.get_id()))?;
+        let max_allowed_outputs_total = match block.header().consensus_data() {
+            ConsensusData::None | ConsensusData::PoW(_) => {
+                amount_sum!(inputs_total, block_subsidy_at_height.0, total_fees.0)
+            }
+            ConsensusData::PoS(_) => {
+                amount_sum!(inputs_total, total_fees.0)
```

Comment:
correct, the subsidy goes to the pool balance and not output

Reply:
I'm not sure I understand the reasoning. There will be duplication, but this isn't a valid proposal.

Reply:
My understanding was that subsidy goes to the pool balance, this way it can be later distributed among staker and delegators according to the rules. So only on decommission or "spend_delegation" will this amount be added to the output. 
Meaning right now `StakePool/ProduceBlock` outputs will be `original amount + fees` and `DecomissionPool` becomes `original amount + accumulated fees + accumulated subsidy`. Later we can apply formula that accounts delegators.
Otherwise if added to the output how can it be later distributed? Isn't it a double spend?

Reply:
@TheQuantumPhysicist please take a look at the comment above

Reply:
just for the history, we decided to revert this change and leave subsidy in the utxo for the consistency reasons and easy access on the decommission